src/net/sourceforge/kolmafia/objectpool/SkillPool.java

```diff
@@ -132,6 +132,8 @@ public class SkillPool {
   public static final int OLD_SCHOOL_COCKTAILCRAFTING = 230;
   public static final int HOLIDAY_MULTITASKING = 240;
   public static final int OVERCLOCK10 = 243;
+  public static final int GENERATE_IRONY = 245;
+
```

Comment:
I assume this is one of those newlines IntelliJ makes you think you should add because of weird spacing. Or something totally different

Reply:
no, it's because I find myself going to the end of 200 and 7000 a lot, so I want to add a newline so I can get there easier (i.e. in vim it's pressing `}` a few times)

Reply:
👍